bazarr/api/episodes/history.py

```diff
@@ -130,24 +128,36 @@ def get(self):
             'matches': x.matched,
             'dont_matches': x.not_matched,
             'external_subtitles': [y[1] for y in ast.literal_eval(x.external_subtitles) if y[1]],
-            'upgradable': bool(x.upgradable) if _language_still_desired(x.language, x.profileId) else False,
             'blacklisted': bool(x.blacklisted),
         } for x in database.execute(stmt).all()]
 
         for item in episode_history:
-            original_video_path = item['path']
-            original_subtitle_path = item['subtitles_path']
+            still_desired = True
+            if not _language_still_desired(item['language'], item['profileId']):
+                # this language isn't desired anymore so there's no point in upgrading this subtitles
+                still_desired = False
```

Comment:
This is pure preference, but _language_still_desired always return a boolean, can we assign it directly?

```suggestion
            still_desired = _language_still_desired(item['language'], item['profileId']):
```

Reply:
I've implemented a custom version of your suggestion keeping the comment. Thanks!